gp-includes/routes/translation.php

```diff
@@ -315,6 +317,12 @@ public function translations_post( $project_path, $locale_slug, $translation_set
 				$translations = GP::$translation->for_translation( $project, $translation_set, 'no-limit', array('translation_id' => $translation->id), array() );
 
 				if ( ! empty( $translations ) ) {
+					if ( $glossary ) {
+						require_once GP_TMPL_PATH . 'helper-functions.php';
```

Comment:
Since `map_glossary_entries_to_translations_originals()` is currently a template function there's no guarantee that the function exists. Maybe it shouldn't be a template function? Would `gp-includes/template.php` be a better place?